data/data/aws/bootstrap/main.tf

```diff
@@ -128,6 +128,10 @@ resource "aws_instance" "bootstrap" {
   }
 
   volume_tags = "${var.tags}"
+
+  credit_specification {
+    cpu_credits = "standard"
```

Comment:
I would recommend unlimited here. I don't know we're optimizing purely for cost. Developer productivity and cluster health are critically important right now. If we exhaust credits during startup of the node and operators, we don't want it to take excessively long to get a stable cluster due to throttling. Also, if we're constantly out of credits for clusters over time, the c5.large is likely a better selection. This assumes this is the correct entry form-factor (2 CPUs and 4 GB memory); we should optimize for performance density of the node type selected.

Reply:
For our masters in OSO starter, we run 3x r4.4xlarge nodes. Customizing these node types will be a common need for the target size of the cluster. Ours stay around 40% CPU utilized. We can send you other statistics/metrics if you are interested.

Reply:
> Customizing these node types will be a common need for the target size of the cluster.

Users can customize already, e.g. by setting [type][1] in their install-config during [a multi-stage install][2].  What's currently missing are knobs in the cluster-API types for detailed tuning like this CPU-credit handling.

[1]: https://github.com/openshift/installer/blob/v0.4.0/pkg/types/aws/machinepool.go#L12-L14
[2]: https://github.com/openshift/installer/blob/v0.4.0/docs/user/overview.md#multiple-invocations

Reply:
Ok, I've rebased onto master and dropped the `cpu_credits` block with 7d236d0 -> 3bdf83b.  I've tried to summarize the motivation for leaving Unlimited mode on towards the end of the 3bdf83b commit message.